pkgs/applications/misc/deco/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, scsh, feh, xlibs }:
+
+stdenv.mkDerivation rec {
+  pname = "deco";
+  version = "0.0.1";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "ebzzry";
+    repo = pname;
+    rev = "18d44aa24eb806ea4c6cbddac14c31bda6d56c2e";
+    sha256 = "0mhb7f4nkv85rn0i03mxszm15hnl0vx1icix3kw7paw69h3i0g0s";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ${pname} $out/bin
+    chmod +x $out/bin/${pname}
+  '';
+
+  postPatch = ''
```

Comment:
It's somewhat more idiomatic to do this in `postFixup` (I know I orginally said to use postPatch), as in `substituteInPlace $out/bin/deco --replace feh "${feh}/bin/feh"`, but it makes no practical difference in this case. Referring to `pname` is a bit unusual, but again makes no practical difference.